dashboard/lib/pd/payment/workshop_summary.rb

```diff
@@ -46,13 +46,13 @@ def qualified?
       !payment.nil?
     end
 
-    def section_url
-      return nil unless workshop.section
-      CDO.code_org_url("/teacher-dashboard#/sections/#{workshop.section.id}", 'http:')
+    def attendance_url
+      return nil if workshop.sessions.empty?
+      "#{workshop_url}/attendance/#{workshop.sessions.first.id}"
```

Comment:
You could write this as a ternary - `workshop.sessions.any? ? "that_string" : nil`